src/Persistent_cohomology/test/persistent_cohomology_unit_test.cpp

```diff
@@ -146,54 +146,54 @@ void test_rips_persistence_in_dimension(int dimension) {
   std::clog << "str_rips_persistence=" << str_rips_persistence << std::endl;
 }
 
-BOOST_AUTO_TEST_CASE( rips_persistent_cohomology_single_field_dim_0 )
+BOOST_AUTO_TEST_CASE( rips_persistent_cohomology_single_field_coeff_0 )
```

Comment:
Here maybe we could make tests more explicit like that:
```cpp
BOOST_AUTO_TEST_CASE( rips_persistent_cohomology_single_field_coeff_not_prime )
{
  for (auto unprime : {0,1,4,6})
    BOOST_CHECK_THROW(test_rips_persistence_with_coeff_field(unprime), std::invalid_argument);
}

BOOST_AUTO_TEST_CASE( rips_persistent_cohomology_single_field_coeff_prime )
{
  for (auto prime : {2,3,5,11,13})
    test_rips_persistence_with_coeff_field(prime);
}

BOOST_AUTO_TEST_CASE( rips_persistent_cohomology_single_field_coeff_limit )
{
  BOOST_CHECK_THROW(test_rips_persistence_with_coeff_field(46349), std::invalid_argument);
}
```

Reply:
Done !